shared/app.js

```diff
@@ -73,7 +73,7 @@ module.exports = Backbone.Model.extend({
      * Initialize the `ClientRouter` on the client-side.
      */
     if (!isServer) {
-      var ClientRouter = this.options.ClientRouter || require(defaultRouterModule);
+      var ClientRouter = this.options.ClientRouter || this.modelUtils._requireAMD(defaultRouterModule);
```

Comment:
Main problem with the signature – array + callback vs string + return.

@purusho Did it work for you in `1.0.1`?